x-pack/plugins/alerts/README.md

```diff
@@ -124,6 +127,18 @@ For example, if the `context` has one variable `foo` which is an object that has
 }
 ```
 
+## Licensing
+
+Currently most of the alerts are free features. But some alert types are subscription features, such as the tracking containment alert.
+
+## Documentation
+
+You should create documentation for the new alert type. Make an entry in the alert type index [`docs/user/alerting/alert-types.asciidoc`](../../../docs/user/alerting/alert-types.asciidoc) that points to a new document for the alert type that should be in the directory [`docs/user/alerting/alert-types`](../../../docs/user/alerting/alert-types).
+
```

Comment:
Presumably solution specific alert types should be under their directories rather than `docs/user/alerting/alert-types/`? 🤔 

If so, best to clarify that

Reply:
I will replace this part with more generic definition. 👍